extensions/3DTILES_metadata/schema/group.schema.json

```diff
@@ -10,9 +10,22 @@
         }
     ],
     "properties": {
+        "id": {
+            "type": "string",
+            "minLength": 1,
+            "description": "Unique identifier for the group."
+        },
+        "name": {
+            "type": "string",
+            "minLength": 1,
+            "description": "The name of the group, e.g. for display purposes."
+        },
```

Comment:
`name` and `description` were removed from metadata entities in https://github.com/CesiumGS/3d-tiles/pull/524.

Reply:
Hm this is a bit of an edge case.. no other metadata entities have an id property (unless they use the `ID` semantic which doesn't have the alphanumeric requirement). This might be an inconsistency we'll have to think more about.

Reply:
I might have misunderstood the statement from

> Slightly orthogonal - but I wonder if the top level groups should be an array instead of a dictionary, and references should be indices instead of strings **(each group definition would have an id property).**

<sup>Emphasis by me</sup>

from https://github.com/CesiumGS/3d-tiles/issues/637#issuecomment-1049005959 - what did this refer to if not an `id` property like the one above?

Reply:
Oh I didn't meant to say this PR was doing anything unexpected.

I'm just rethinking my original suggestion to add an `id` since no other metadata entities have an `id` and I don't think groups are necessarily a special case.

Reply:
Yeah, one could argue a bit here: 

Q: "Which tile is this?" 
A : "The tile with url `example.com/t.b3dm`"

vs. 

Q: "Which group is this?"  
A: 42!

Groups are "special" in that they do not have any representation in the _structure_. One of their main purposes is giving a _meaning_ to an otherwise random collection of things. I'd consider that a reason why giving them an _id_ as well as a _name_ and _description_ may be more justified.

Reply:
`id`, `name`, `description` could be given by metadata semantics. It depends if we want to enforce `id` or not.